mmv1/third_party/terraform/services/compute/data_source_google_compute_region_backend_service.go

```diff
@@ -0,0 +1,56 @@
+package compute
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func DataSourceGoogleComputeRegionBackendService() *schema.Resource {
+	// Get schema from the resource
+	dsSchema := tpgresource.DatasourceSchemaFromResourceSchema(ResourceComputeRegionBackendService().Schema)
+
+	// Set 'Required' schema elements
+	tpgresource.AddRequiredFieldsToSchema(dsSchema, "name")
+	tpgresource.AddRequiredFieldsToSchema(dsSchema, "region")
```

Comment:
I think we may want to set region as optional,  so it could take provider region, also given that we have `region, err := tpgresource.GetRegion(d, config)` in read